### PR TITLE
cloud: add msp docs

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC-tidb-cloud.md [hunk 1]
```diff
@@ -284,6 +284,9 @@
   - [Billing from AWS or GCP Marketplace](/tidb-cloud/tidb-cloud-billing.md#billing-from-aws-marketplace-or-google-cloud-marketplace)
   - [Billing for Changefeed](/tidb-cloud/tidb-cloud-billing-ticdc-rcu.md)
   - [Billing for Data Migration](/tidb-cloud/tidb-cloud-billing-dm.md)
+- Managed Service Provider
+  - [MSP Partner](/tidb-cloud/managed-service-provider-partner.md)
+  - [MSP Customer](/tidb-cloud/managed-service-provider-customer.md)
```
Comment (ran-huang) [inline TOC-tidb-cloud.md:289]: api url is added to TOC in https://github.com/pingcap/docs/pull/16040

File: tidb-cloud/managed-service-provider-customer.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+# Managed Service Provider Customer
+
+A managed service provider (MSP) is an end customer who purchases TiDB Cloud and pays invoices through the MSP channel.
+
+This document describes how to become an MSP customer and some common operations for MSP customers.
+
+## Create a new MSP customer account
+
+To create a new MSP customer account, visit the MSP dedicated sign-up page. Each MSP has their own dedicated sign-up page, ask your MSP for the sign-up page URL.
```
Comment (hfxsd) [inline tidb-cloud/managed-service-provider-customer.md:14]:
```suggestion
To create a new MSP customer account, visit the MSP dedicated sign-up page. Each MSP has a unique dedicated sign-up page. Contact your MSP to get the sign-up page URL.
```

File: tidb-cloud/managed-service-provider-partner.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: Managed Service Provider Partner
+summary: Learn how to become a Managed Service Provider Partner.
```
Comment (hfxsd) [inline tidb-cloud/managed-service-provider-partner.md:3]:
```suggestion
summary: Learn how to become a Managed Service Provider partner.
```

File: tidb-cloud/tidb-cloud-glossary.md [hunk 4]
```diff
@@ -67,6 +71,14 @@ A user that has been invited to an organization, with access to the organization
 
 Starting from v5.0, TiDB introduces Massively Parallel Processing (MPP) architecture through TiFlash nodes, which shares the execution workloads of large join queries among TiFlash nodes. When the MPP mode is enabled, TiDB, based on cost, determines whether to use the MPP framework to perform the calculation. In the MPP mode, the join keys are redistributed through the Exchange operation while being calculated, which distributes the calculation pressure to each TiFlash node and speeds up the calculation. For more information, see [Use TiFlash MPP Mode](/tiflash/use-tiflash-mpp-mode.md).
 
+### MSP Customer
+
+A managed service provider (MSP) customer is an end customer who purchases TiDB Cloud and pays invoices through the MSP channel. This is distinguished from a [direct customer](#direct-customer).
+
+### MSP Partner
+
+A managed service provider (MSP) is a partner who resells TiDB Cloud and provides value-added services, including but not limited to TiDB Cloud organization management, billing services, and technical support.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-glossary.md:80]:
```suggestion
A managed service provider (MSP) partner is a partner who resells TiDB Cloud and provides value-added services, including but not limited to TiDB Cloud organization management, billing services, and technical support.
```

File: tidb-cloud/managed-service-provider-customer.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: Managed Service Provider Customer
+summary: Learn how to become a Managed Service Provider Customer.
```
Comment (hfxsd) [inline tidb-cloud/managed-service-provider-customer.md:3]:
```suggestion
summary: Learn how to become a Managed Service Provider customer.
```

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/managed-service-provider-customer.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+# Managed Service Provider Customer
+
+A managed service provider (MSP) is an end customer who purchases TiDB Cloud and pays invoices through the MSP channel.
```
Comment (hfxsd) [inline tidb-cloud/managed-service-provider-customer.md:8]:
```suggestion
A managed service provider (MSP) customer is an end customer who purchases TiDB Cloud and pays invoices through the MSP channel.
```

File: tidb-cloud/tidb-cloud-glossary.md [hunk 7]
```diff
@@ -67,6 +71,14 @@ A user that has been invited to an organization, with access to the organization
 
 Starting from v5.0, TiDB introduces Massively Parallel Processing (MPP) architecture through TiFlash nodes, which shares the execution workloads of large join queries among TiFlash nodes. When the MPP mode is enabled, TiDB, based on cost, determines whether to use the MPP framework to perform the calculation. In the MPP mode, the join keys are redistributed through the Exchange operation while being calculated, which distributes the calculation pressure to each TiFlash node and speeds up the calculation. For more information, see [Use TiFlash MPP Mode](/tiflash/use-tiflash-mpp-mode.md).
 
+### MSP Customer
+
+A managed service provider (MSP) customer is an end customer who purchases TiDB Cloud and pays invoices through the MSP channel. This is distinguished from a [direct customer](#direct-customer).
+
+### MSP Partner
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-glossary.md:78]: MSP partner 已经删掉了吗？这一条要不要改成 MSP

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/tidb-cloud-glossary.md (hunk 7 above)
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-glossary.md:78]:
```suggestion
### Managed Service Provider (MSP)
```

File: tidb-cloud/tidb-cloud-glossary.md [hunk 8]
```diff
@@ -67,6 +71,14 @@ A user that has been invited to an organization, with access to the organization
...
+
+A managed service provider (MSP) partner is a partner who resells TiDB Cloud and provides value-added services, including but not limited to TiDB Cloud organization management, billing services, and technical support.
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-glossary.md:80]:
```suggestion
A managed service provider (MSP) is a partner who resells TiDB Cloud and provides value-added services, including but not limited to TiDB Cloud organization management, billing services, and technical support.
```

File: tidb-cloud/managed-service-provider-customer.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+## Check your future bills
+
+Once you successfully become an MSP customer, you will pay invoices through your MSP. Ask your MSP where you can check your bills and invoices.
+
+PingCAP does not send any bills and invoices to MSP customers.
+
+## Check your history bills
+
+If you have migrated from a direct TiDB Cloud account to an MSP customer account, you can view your history bills prior to the migration by visiting **Billing** > **Bills** in the TiDB Cloud console.
```
Comment (ran-huang) [inline tidb-cloud/managed-service-provider-customer.md:36]:
```suggestion
If you have migrated from a direct TiDB Cloud account to an MSP customer account, you can view your history bills prior to the migration by visiting **Billing** > **Bills** > **History** in the TiDB Cloud console.
```

File: TOC-tidb-cloud.md [hunk 10]
```diff
@@ -284,6 +284,9 @@
   - [Billing from AWS or GCP Marketplace](/tidb-cloud/tidb-cloud-billing.md#billing-from-aws-marketplace-or-google-cloud-marketplace)
   - [Billing for Changefeed](/tidb-cloud/tidb-cloud-billing-ticdc-rcu.md)
   - [Billing for Data Migration](/tidb-cloud/tidb-cloud-billing-dm.md)
+- Managed Service Provider
+  - [MSP Partner](/tidb-cloud/managed-service-provider-partner.md)
```
Comment (ran-huang) [inline TOC-tidb-cloud.md:288]:
```suggestion
  - [Managed Service Provider](/tidb-cloud/managed-service-provider.md)
```

**Review by zhoubasten — APPROVED** [review]
LGTM

**Pull request merged**